cypress.json

```diff
@@ -4,7 +4,7 @@
   "viewportHeight": 800,
   "defaultCommandTimeout": 40000,
   "pageLoadTimeout": 120000,
-  "video": false,
+  "video": true,
```

Comment:
idk why, but locally this fixed the issue in headless mode for me... let's see if it works on ci as well